src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -313,6 +336,12 @@ export class PatientViewPageStore {
             ),
     });
 
+    readonly discreteMolecularProfile = remoteData({
```

Comment:
couldn't there be more than one discrete molecular profile?

Reply:
There could be, but we already have `readonly molecularProfileIdDiscrete`. I just followed the same logic.